noir-projects/aztec-nr/aztec/src/utils/point.nr

```diff
@@ -14,7 +14,7 @@ pub fn point_to_bytes(pk: Point) -> [u8; 32] {
     let mut result = pk.x.to_be_bytes(32);
 
     // We store only a "sign" of the y coordinate because the rest can be derived from the x coordinate. To get
-    // the sign we check if the y coordinate is greater than the curve's order minus 1 divided by 2.
+    // the sign we check if the y coordinate is less or equal than the curve's order minus 1 divided by 2.
     if !BN254_FR_MODULUS_DIV_2.lt(pk.y) {
```

Comment:
```suggestion
    // the sign we check if the y coordinate is less or equal than the curve's order minus 1 divided by 2.
    // Ideally we'd do `y <= MOD_DIV_2`, but there's no `lte` function, so instead we do `!(y > MOD_DIV_2)`, which is
    // equivalent, and then rewrite that as `!(MOD_DIV_2 < y)`, since we also have no `gt` function.
    if !BN254_FR_MODULUS_DIV_2.lt(pk.y) {
```

Great catch! Left a small comment as I find this `if` very cryptic.